packages/node/src/indexer/sandbox.ts

```diff
@@ -23,7 +23,7 @@ const DEFAULT_OPTION: NodeVMOptions = {
   wasm: false,
   sandbox: {},
   require: {
-    builtin: ['assert'],
+    builtin: ['assert', 'buffer', 'crypto', 'util', 'events', 'path'],
```

Comment:
events is not supported by vm2, https://github.com/patriksimek/vm2/issues/216#issuecomment-528866482

Reply:
let's drop it for now.